src/OrleansCodeGenerator/TaskUtility.cs

```diff
@@ -0,0 +1,111 @@
+﻿/*
+Project Orleans Cloud Service SDK ver. 1.0
+ 
+Copyright (c) Microsoft Corporation
+ 
+All rights reserved.
+ 
+MIT License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and 
+associated documentation files (the ""Software""), to deal in the Software without restriction,
+including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so,
+subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+namespace OrleansCodeGenerator
+{
+    using System;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Extension methods for <see cref="Task"/>.
+    /// </summary>
+    public static class TaskUtility
+    {
+        /// <summary>
+        /// Returns a faulted task.
+        /// </summary>
+        /// <param name="exception">The exception which the return task faulted with.</param>
+        /// <returns>A faulted task.</returns>
+        public static Task<object> Faulted(Exception exception)
```

Comment:
That's fine. We could move it to PublicOrleansTaskExtentions, but that will expose it to app code, which may not be desirable.